qark/qarkMain.py

```diff
@@ -410,15 +411,15 @@ def setup_argparse():
     return parser
 
 
-def run_automated_defaults(pathToReport, pathToApk):
+def run_automated_defaults(pathToApk, pathToReport):
```

Comment:
The parameter order was reversed from what is expected in run_automated_defaults